### PR TITLE
Added testnet 48

### PR DESCRIPTION
**Pull request by janezpodhostnik**
## Description

Added testnet 48 to sporks.json

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

**Review by zhangchiqing — APPROVED** [review]

File: sporks.json [hunk 1]
```diff
@@ -1309,6 +1309,45 @@
       }
     },
     "testnet": {
+      "testnet48": {
+        "id": 48,
+        "live": true,
+        "name": "Testnet48",
+        "sporkTime": "2023-10-25T14:30:00Z",
+        "rootHeight": "127720465",
```
Comment (peterargue) [inline sporks.json:1317]: this should be `127720466` according to the root snapshot
https://storage.googleapis.com/flow-genesis-bootstrap/testnet-48/public-root-information/root-protocol-state-snapshot.json

File: sporks.json [hunk 2]
```diff
@@ -1309,6 +1309,45 @@
...
+          "flow-dps-tag": "v0.32.3",
+          "cadence-tag": "v0.42.1",
+          "docker-tag": "v0.32.3"
+        },
+        "accessNodes": [
+          "access-001.devnet48.nodes.onflow.org:9000"
+        ],
+        "seedNodes": [
+          {
+            "address": "access-001.devnet48.nodes.onflow.org:3570",
```
Comment (peterargue) [inline sporks.json:1346]: this should be the access-003 node. we just transition it to 001 for past sporks.
```suggestion
            "address": "access-003.devnet48.nodes.onflow.org:3570",
```


this is confusing and I'll look to just switch it all over to use AN1 in the future

File: sporks.json [hunk 3]
```diff
@@ -1309,6 +1309,45 @@
       }
     },
     "testnet": {
+      "testnet48": {
+        "id": 48,
+        "live": true,
+        "name": "Testnet48",
+        "sporkTime": "2023-10-25T14:30:00Z",
```
Comment (gomisha) [inline sporks.json:1316]: Spork time on root snapshot is `2023-10-25T15:21:04Z`

https://storage.googleapis.com/flow-genesis-bootstrap/testnet-48/public-root-information/root-protocol-state-snapshot.json

File: sporks.json [hunk 4]
```diff
@@ -1309,6 +1309,45 @@
...
+          "cadence-tag": "v0.42.1",
+          "docker-tag": "v0.32.3"
+        },
+        "accessNodes": [
+          "access.devnet.nodes.onflow.org:9000"
+        ],
+        "seedNodes": [
+          {
+            "address": "access-003.devnet48.nodes.onflow.org:3570",
+            "key": "ba69f7d2e82b9edf25b103c195cd371cf0cc047ef8884a9bbe331e62982d46daeebf836f7445a2ac16741013b192959d8ad26998aff12f2adc67a99e1eb2988d"
```
Comment (peterargue) [inline sporks.json:1347]: this should be the original access-003 key.
```suggestion
            "key": "b662102f4184fc1caeb2933cf87bba75cdd37758926584c0ce8a90549bb12ee0f9115111bbbb6acc2b889461208533369a91e8321eaf6bcb871a788ddd6bfbf7"
```

Comment (janezpodhostnik) [reply sporks.json]: I see. Changed, please check again

**Review by peterargue — APPROVED** [review]

**Pull request merged**